src/components/MultiHopTrade/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -21,7 +23,7 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = ({ isOpen }) => {
     if (!quote) return null
 
     // TODO(woodenfurniture): use quote ID when we want to support multiple quotes per swapper
-    const isActive = activeSwapperName === swapperName
+    const isActive = selectedSwapperName === swapperName
```

Comment:
Curious what's the rationale behind the name change here? IMO, `active` carries the meaning of state-active, while `selected` carries UI-only vernacular, i.e a swapper might've been `selected` from a UI standpoint, but not yet reconciliated as `active`.

Reply:
The architecture is still in the air but as it stands the selected quote and swapper are only ever active, i.e its not surrently possible to have a selected swaper without a corresponding quote.

That said, active does make more sense here such that if the architecture did change (there is a PR on the way to change it), the vernacular would stand as valid.

will change